extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -115,11 +126,21 @@ private void enableKubernetes(ApplicationInfoBuildItem applicationInfo, Kubernet
         }
 
         kubernetesProperties.put("ports", toArray(ports));
+        if (httpLivenessProbePath.isPresent()) {
+            Map<String, Object> livenessProbeProperties = new HashMap<>();
+            livenessProbeProperties.put("httpActionPath", httpLivenessProbePath.get());
+            kubernetesProperties.put("livenessProbe", livenessProbeProperties);
+        }
+        if (httpReadinessProbePath.isPresent()) {
+            Map<String, Object> readinessProbeProperties = new HashMap<>();
+            readinessProbeProperties.put("httpActionPath", httpReadinessProbePath.get());
+            kubernetesProperties.put("readinessProbe", readinessProbeProperties);
+        }
         if (httpProbePath.isPresent()) {
```

Comment:
I think I would get rid of this one.

Reply:
OK, even if we keep the root probe, I think it does not make sense to have it here.

@geoand do you agree?

Reply:
Give the fact that the other will always be set, yeah. How about we take it a step further and drop `kubernetesHealthPathBuildItem` completely?

Reply:
Yes, that's the obvious consequence :).